android/src/main/java/com/reactlibrary/rnwifi/RNWifiModule.java

```diff
@@ -576,4 +568,20 @@ private void stuffWifiConfigurationWithoutEncryption(final WifiConfiguration wif
     private String formatWithBackslashes(final String value) {
         return String.format("\"%s\"", value);
     }
+
+    private boolean isLocationPermissionGranted(final Promise promise) {
```

Comment:
Not a 100% fan of the method name, as this method does the check _and_ rejects the promise, which isn't clear now.

Reply:
maybe `assertLocationPermissionGranted` or `rejectWhenLocationPermissionMissing`?